pkg/sdkserver/sdkserver_test.go

```diff
@@ -700,6 +699,8 @@ func TestSDKServerWatchGameServerFeatureSDKWatchSendOnExecute(t *testing.T) {
 	stop := make(chan struct{})
 	defer close(stop)
 	sc.informerFactory.Start(stop)
+
+	fakeWatch.Add(fixture.DeepCopy())
```

Comment:
Using a fake Watch so we can control explicitly when the game server is modified.